controllers/tenants.rb

```diff
@@ -7,3 +8,9 @@
 get '/apartments/:id/tenants/new' do
   erb :"tenants/new"
 end
+
+# create
+post "/apartments" do
```

Comment:
instead of posting to apartments, you should:

```rb
post "/apartments/:id/tenants"
  # so you know which apartment to add the tenant to.
end
```